### ISSUE TITLE
Vanilla names that fit in the battle UI incorrectly use a narrow font

### ISSUE THREAD

**Issue by Sneed69** [rh-hideout/pokeemerald-expansion#4712]
### Description

![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/56992013/1e094a4d-156f-44a4-a9ee-84d8b6234f30)

### Version

upcoming (Edge)

### Discord contact info

duke5614

**Comment by pkmnsnfrn** [comment]
@AsparagusEduardo I agree this isn't GREAT, but I don't think its simple OR degrading the experience. Do you truly believe this should block 1.9.0.?

**Comment by AsparagusEduardo** [comment]
> @AsparagusEduardo I agree this isn't GREAT, but I don't think its simple OR degrading the experience. Do you truly believe this should block 1.9.0.?

I can't imagine this fix would be hard enough to implement (famous last words ^^;)

**Comment by mrgriffin** [comment]
It may be possible to add some tests which check all the vanilla names fit in all the spaces? Those tests would look similar to the existing ones (in `test/text.c`), but use the default font for the particular UIs instead of the narrowest ones and only test the vanilla IDs.
